### PR TITLE
Fix: View facility styling

### PR DESCRIPTION
**Pull request by punwai**
**Link**: Admin->User->View Facility->’Click on user’s name’

**Work**: - add some space between “.... Information” and “Back to users” button
-Separate “Save” and “Cancel” buttons
Space between fields`
Bring fields closer to their labels.

**FIXED:**
- add some space between “.... Information” and “Back to users” button
- Separate “Save” and “Cancel” buttons
- Space between fields
- Bring fields closer to their labels.

**From this**
screen shot 2018-12-09 at 00 07 41
screen shot 2018-12-08 at 05 47 20

**To this:**
screen shot 2018-12-09 at 00 00 06

### CODE REVIEW

File: interface/usergroup/facility_user_admin.php [hunk 1]
```diff
@@ -123,6 +123,16 @@ function trimAll(sString)
...
+<style>
+    .large_button{
+        margin-bottom: 5px;
+        float: right;
+    }
+    td {
+        padding-bottom: 10px;
+    }
+</style>
+
```
Comment (robbyoconnor) [inline interface/usergroup/facility_user_admin.php]: This should go in CSS files.

File: interface/usergroup/facility_user_admin.php [hunk 2]
```diff
@@ -143,7 +153,7 @@ function trimAll(sString)
     <input type=hidden name=fac_id value="<?php echo attr($_GET["fac_id"]);?>">
     <?php $iter = sqlQuery("select * from facility_user_ids where id=?", array($my_id)); ?>
 
-<table border=0 cellpadding=0 cellspacing=0>
+<table style="width: 50%; min-width: 150px; max-width: 250px" border=0 cellpadding=0 cellspacing=0>
```
Comment (robbyoconnor) [inline interface/usergroup/facility_user_admin.php]: Put in a CSS file.

File: interface/usergroup/facility_user.php [hunk 3]
```diff
@@ -130,7 +130,7 @@
        <table>
       <tr >
         <td><b><?php echo xlt('Facility Specific User Information'); ?></b></td>
-        <td><a href="usergroup_admin.php" class="css_button cp-misc" onclick="top.restoreSession()"><span><?php echo xlt('Back to Users'); ?></span></a>
+        <td><a style="margin-left: 30px;" href="usergroup_admin.php" class="css_button cp-misc" onclick="top.restoreSession()"><span><?php echo xlt('Back to Users'); ?></span></a>
```
Comment (robbyoconnor) [inline interface/usergroup/facility_user.php]: put in a CSS file.

**Comment by punwai** [comment]
The buttons should not have blue text, it is part of a theme which provides blue text.

**Pull request merged**